pkg/agent/apiserver/handlers/fqdncache/handler_test.go

```diff
@@ -26,48 +26,67 @@ import (
 	"github.com/stretchr/testify/require"
 	"go.uber.org/mock/gomock"
 
+	"antrea.io/antrea/pkg/agent/apis"
 	"antrea.io/antrea/pkg/agent/types"
 	queriertest "antrea.io/antrea/pkg/querier/testing"
 )
 
 func TestFqdnCacheQuery(t *testing.T) {
 	tests := []struct {
-		name             string
-		expectedStatus   int
-		expectedResponse []types.DnsCacheEntry
+		name                 string
+		expectedStatus       int
+		filteredCacheEntries []types.DnsCacheEntry
+		expectedResponse     []apis.FQDNCacheResponse
 	}{
 		{
 			name:           "FQDN cache exists - multiple addresses multiple domains",
 			expectedStatus: http.StatusOK,
-			expectedResponse: []types.DnsCacheEntry{
+			filteredCacheEntries: []types.DnsCacheEntry{
 				{
-					FqdnName:       "example.com",
-					IpAddress:      net.ParseIP("10.0.0.1"),
+					FQDNName:       "example.com",
+					IPAddress:      net.ParseIP("10.0.0.1"),
 					ExpirationTime: time.Date(2025, 12, 25, 15, 0, 0, 0, time.UTC),
```

Comment:
Since the same time is used everywhere, I suggest you declare one variable of type `time.Time` at the beginning of the test function, which you can then use everywhere. Actually, we usually do something like this:
```go
expirationTime := time.Now().Add(1*time.Hour)
```
then use `ExpirationTime: expirationTime` everywhere